ts/src/bybit.ts

```diff
@@ -1766,6 +1711,192 @@ export default class bybit extends Exchange {
+                    if (rawMarketsLength === 0) {
+                        break;
+                    }
+                    markets = this.arrayConcat (rawMarkets, markets);
+                    paginationCursor = this.safeString (dataNew, 'nextPageCursor');
+                }
+            }
+        }
+        //
+        //     {
+        //         "retCode": 0,
+        //         "retMsg": "success",
+        //         "result": {
+        //             "category": "option",
+        //             "nextPageCursor": "0%2C2",
+        //             "list": [
+        //                 {
+        //                     "symbol": "BTC-29DEC23-80000-C",
+        //                     "status": "Trading",
+        //                     "baseCoin": "BTC",
+        //                     "quoteCoin": "USD",
+        //                     "settleCoin": "USDC",
+        //                     "optionsType": "Call",
+        //                     "launchTime": "1688630400000",
+        //                     "deliveryTime": "1703836800000",
+        //                     "deliveryFeeRate": "0.00015",
+        //                     "priceFilter": {
+        //                         "minPrice": "5",
+        //                         "maxPrice": "10000000",
+        //                         "tickSize": "5"
+        //                     },
+        //                     "lotSizeFilter": {
+        //                         "maxOrderQty": "500",
+        //                         "minOrderQty": "0.01",
+        //                         "qtyStep": "0.01"
+        //                     }
+        //                 },
+        //             ]
+        //         },
+        //         "retExtInfo": {},
+        //         "time": 1688873094448
+        //     }
+        //
+        const result = [];
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            const id = this.safeString (market, 'symbol');
+            const baseId = this.safeString (market, 'baseCoin');
+            const quoteId = this.safeString (market, 'quoteCoin');
+            const settleId = this.safeString (market, 'settleCoin');
+            const base = this.safeCurrencyCode (baseId);
+            const quote = this.safeCurrencyCode (quoteId);
+            const settle = this.safeCurrencyCode (settleId);
+            const lotSizeFilter = this.safeValue (market, 'lotSizeFilter', {});
+            const priceFilter = this.safeValue (market, 'priceFilter', {});
+            const status = this.safeString (market, 'status');
+            const expiry = this.safeInteger (market, 'deliveryTime');
+            const splitId = id.split ('-');
+            const strike = this.safeString (splitId, 2);
+            const optionLetter = this.safeString (splitId, 3);
+            if ((this.options['loadAllOptions']) || (this.options['loadExpiredOptions'])) {
```

Comment:
hum probably we don't need this repetitive if-else clause, we can do something like

`if (status === 'Trading') || (this.options['loadAllOptions']) || (this.options['loadExpiredOptions']`